src/templates/cart/voucher.template.html

```diff
@@ -124,7 +122,7 @@
 <hr />
 <div>
 <a name="voucher[@reward_id@]"></a>
-<h4>#[@item_count@] &nbsp; [@vprogram_title@] - 
+<h4>#[@item_count@] &nbsp; [@vprogram_title@] -
```

Comment:
This change would actually flush the points right up against the hyphen. We will need to change it so the logic that follows is on the same line as this. Had may as well update to the new syntax too.